ui/packages/components/src/types/workers.ts

```diff
@@ -13,7 +13,7 @@ export const convertWorkerStatus = (status: WorkerStatus): GroupedWorkerStatus |
     case 'READY':
       return 'ACTIVE';
     case 'DISCONNECTED':
-      return 'FAILED';
+      return 'DISCONNECTED';
```

Comment:
Have we moved away from 
<img width="160" alt="Screenshot 2025-01-28 at 12 34 20" src="https://github.com/user-attachments/assets/2d0b98b5-17b9-4459-9f6a-b35593040845" />
and are we replacing failed by disconnected (with grey instead of red)? I couldn't find where that decision was made on Slack

Reply:
Disconnected and failed are two separate states, workers can disconnect gracefully and that's totally fine and probably shouldn't be marked as a failure. Connections can also fail, which we don't track separately right now but could do in the future.